sklearn/decomposition/_pca.py

```diff
@@ -382,7 +382,7 @@ def fit(self, X, y=None):
         self._fit(X)
         return self
 
-    def fit_transform(self, X, y=None):
+    def fit_transform(self, X, y=None, n_oversamples_rate=0.8):
```

Comment:
This is not following the scikit-learn API. You cannot pass this parameter here. It should be given during `__init__`.

Reply:
Thank you, according to your suggestion, we have completed the modification.